browser_patches/firefox/patches/bootstrap.diff

```diff
@@ -2024,9 +2024,14 @@ index 0000000000000000000000000000000000000000..b8a98b058ea36f85fddfa21e992e2d06
 +    if (this._disposed)
 +      return;
 +
-+    const body = this._chunks.join('');
++    if (aStatusCode === 0) {
```

Comment:
Just to double check: this is not an http status code?

Reply:
If you meant http response code then it's not, it's a mozilla internal status code (nsresult).